GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -114,12 +114,15 @@ extension Glia {
         interactor = createdInteractor
 
         if let callback = completion {
-            createdInteractor.withConfiguration { [weak createdInteractor] in
+            createdInteractor.withConfiguration { [weak createdInteractor, weak self] in
                 guard let interactor = createdInteractor else { return }
-                    interactor.state = GliaCore.sharedInstance
-                        .getCurrentEngagement()?.engagedOperator
-                        .map(InteractorState.engaged) ?? interactor.state
-                    callback()
+
+                self?.stringProviding = .init(getRemoteString: GliaCore.sharedInstance.localeProvider.getRemoteString)
+
+                interactor.state = GliaCore.sharedInstance
```

Comment:
[Same](https://github.com/salemove/ios-sdk-widgets/pull/779/files#r1333081122).

Reply:
Fixed 20346c66f050980bcdc2774225a81282b714a7c7

Reply:
Now it's fixed 😄 2550ea25f4220bad6f2283d72d39110572d42692